_posts/2024-01-29-strimzicon2024-announcement.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title:  "Welcome to the StrimziCon 2024!"
+date: 2024-01-29
+author: paolo_patierno
+---
+
+We are very pleased to announce the first StrimziCon coming this year!
+This is going to be the first conference focused on our beloved Strimzi project and it will be "virtual".
+It is organized and driven by the project maintainers together with CNCF.
+
+<!--more-->
+
+![StrimziCon 2024 Banner](/assets/images/posts/2024-01-29-strimzicon2024-banner.png)
+
+The StrimziCon is an event for developers, DevOps engineers, and solution architects who want to learn about Strimzi and event streaming.
+This event will bring together the Strimzi community, and you will learn from experts and users about use cases, best practices, and their experience using Strimzi in their organizations.
```

Comment:
```suggestion
This event will bring together the Strimzi community, where you can learn from experts and users about use cases, best practices, and their experiences using Strimzi in their organizations.

```

Reply:
It's the official description already fixed by different people I would stick with that.